### DISCUSSION TITLE
Mistake in the new installation guide

### DISCUSSION THREAD

**Discussion by kristianinc** [scenerystack/community#76] [SceneryStack]
Hello friends. Since the last installation guide update, I realized that there is a mistake.
The document starts by cloning perennial but then references perennial alias in the next step.
Then, even after getting the perennial alias manually, I was unable to run the .sh due to missing files. which require running npm install in the perennial-alias. from there, everything works pretty well.

**Suggested changes**

- step1 create the phetsims folder
- step2 clone the perennial-alias
- step3 cd into perennial alias and run npm install. and cd back into phetsims.
- step4 run the clone missing-repos step and proceed.

Please let me know in case of any corrections or suggestions.

**Comment by brettfiedler** [comment]
Thanks! That was indeed due to a recent change in perennial-alias that was not caught for fresh installs. 

I made an [issue](https://github.com/scenerystack/community/issues/77) from this and @samreid is making an update to the Development Overview now.

You are welcome to suggest changes to the documentation as well through pull requests if you are so inclined! 

I should note that we're currently working on a new way to work with the SceneryStack libraries through an npm package which should simplify this whole process. And will include a big update to the documentation as well!

**Reply by kristianinc** [reply]
Got it @brettfiedler, thank you.
